File: app/src/main/java/com/example/mettest/models/Datum.java

package com.example.mettest.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Datum {

    @SerializedName("cat_id")
    @Expose
    public String catId;
    @SerializedName("cat_name")
    @Expose
    public String catName;
    @SerializedName("items")
    @Expose
    public List<Item> items = null;

}

File: app/src/main/java/com/example/mettest/models/Datum.kt
package com.example.mettest.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Datum {
    @SerializedName("cat_id")
    @Expose
    var catId: String? = null

    @SerializedName("cat_name")
    @Expose
    var catName: String? = null

    @SerializedName("items")
    @Expose
    var items: List<Item>? = null
}